src/clients/storage/GraphStorageClient.cpp

```diff
@@ -165,7 +165,7 @@ folly::SemiFuture<StorageRpcResponse<cpp2::ExecResponse>> GraphStorageClient::ad
       evb,
       std::move(requests),
       [=](cpp2::GraphStorageServiceAsyncClient* client, const cpp2::AddEdgesRequest& r) {
-        return useToss ? client->future_addEdgesAtomic(r) : client->future_addEdges(r);
+        return useToss ? client->future_chainAddEdges(r) : client->future_addEdges(r);
```

Comment:
In `src/storage/InternalStorageServiceHandler.h ` file, the `future_chainAddEdges ` should be InterStorageServiceAsyncClient?

Reply:
Please ignore this, both `InternalStorageService` and `GraphStorageService` have this function.